src/features/projects/screens/SingleProjectDetails.tsx

```diff
@@ -101,6 +100,8 @@ function SingleProjectDetails({}: Props): ReactElement {
         }`
       );
     } else {
+      const link = localStorage.getItem('redirectLink');
+      const previousPath = link?.replace(link[0], '');
```

Comment:
I visited https://planet-webapp-lc2mzz1t5-planetapp.vercel.app/profile/treemapper/import resulting into storing this value into the locale storage value `redirectLink` (took this as an extreme example to better make my point). When I now visit https://planet-webapp-lc2mzz1t5-planetapp.vercel.app/science-forest-planbe?site=site_aufcHOggZ6setis and click the back link, I get redirected to https://planet-webapp-lc2mzz1t5-planetapp.vercel.app/profile/treemapper/import - I don't think that's what we want.

Reply:
I did this  to redirect the user back to that page from where he visited to  /:projectID